rhythmfpv.md

```diff
@@ -0,0 +1,26 @@
+name: "rhythm-fpv"
+project: "Racing FPV Drone Build"
+
+#rhythmfpv
+# "Racing FPV Drone Build"
+THESE ARE NOT PRE ASSEMBLED KITS, IT REQUIRES A LOT OF EXPERIENCE AND IS A TOUGH, COMPETETIVE AND EXPENSIVE HOBBY WHICH EVERYONE WOULD LOVE 
+(FPV - First Person View)
+## I am building a racing fpv drone. So basically fpv is a whole new hobby in itself. Everyone builds thier own quadcopters does freestyle, The FPV community has recently started to grow in various parts of country.Companies like DJI, caddx, foxeer, TBS are a big contribution to this. New technologies like elrs are evolving and these superfast small compact drones are a huge oppurtunity for someone looking to enter a competetive and challenging hobby. The drone i am planning to make will fit well under 250usd.
+
+## plan
+First of all we require a rigid frame which is made of carbon firbre, we need high rpm motors to make our quad super fast and powerful
+we will require a transmitter and reciever to controll our drone. To run our motors we need Electronic Speed Controller (ESC). To control everything
+we will require the brain of our quadcopter, the flight controller. lastly we will require an fpv system which creates a whole new world for us 
+which includes a small cute camera, a video transmitter and video reciever. we can now see what the drone sees in air and why according to it, thats why
+its called FPV.
+
+## budget
+ | Product         | Supplier/Link                         | Cost   |
+| --------------- | -------------------------------------  | ------ |
+| HIGH KV FPV MOTORS (x4)  | shorturl.at/dlovw             | $40    |
+| Flight Controller and ESC| shorturl.at/cjmxz             | $74    |
+| Carbon Fibre 5inch Frame | shorturl.at/ipuMS             | $13    |
+| Transmitter And Reciever | shorturl.at/knSU8             | $60    |
+| FPV Camera               | shorturl.at/eHNWY             | $19    |
+| Video Rx And transmitter | shorturl.at/aDQV7             | $44    | 
```

Comment:
it looks like this link is also expired or smth as it redirects me to https://www.shorturl.at/